### PR TITLE
Replace usage of set_format_arg by Formatter::Common()

### PR DESCRIPTION
**Pull request by brenoguim**
Part of #11389

### CODE REVIEW

File: src/openrct2/paint/tile_element/Paint.Entrance.cpp [hunk 1]
```diff
@@ -270,20 +271,20 @@ static void park_entrance_paint(paint_session* session, uint8_t direction, int32
                 break;
 
             {
-                set_format_arg(0, uint32_t, 0);
-                set_format_arg(4, uint32_t, 0);
+                Formatter::Common().Add<uint32_t>(0).Add<uint32_t>(0);
```
Comment (tupaschoal) [inline src/openrct2/paint/tile_element/Paint.Entrance.cpp:274]: Should this get the same comment as #11388 ?

File: src/openrct2/paint/tile_element/Paint.LargeScenery.cpp [hunk 2]
```diff
@@ -301,8 +301,7 @@ void large_scenery_paint(paint_session* session, uint8_t direction, uint16_t hei
         }
         // 6B8331:
         // Draw sign text:
-        set_format_arg(0, uint32_t, 0);
-        set_format_arg(4, uint32_t, 0);
+        Formatter::Common().Add<uint32_t>(0).Add<uint32_t>(0);
```
Comment (tupaschoal) [inline src/openrct2/paint/tile_element/Paint.LargeScenery.cpp:304]: Ditto

**Comment by duncanspumpkin** [comment]
For your next pull request could you look into makeing a deep copy function for the formatter. We will need it for future refactorings especially with refactoring the scrolling text.

**Comment by Gymnasiast** [comment]
ping @brenoguim

File: src/openrct2/paint/tile_element/Paint.Entrance.cpp (hunk 1 above)
Comment (brenoguim) [reply src/openrct2/paint/tile_element/Paint.Entrance.cpp:274]: I wouldn't be able to tell. I don't know if the arguments are cleared for the same reason.

File: src/openrct2/paint/tile_element/Paint.LargeScenery.cpp (hunk 2 above)
Comment (brenoguim) [reply src/openrct2/paint/tile_element/Paint.LargeScenery.cpp:304]: I wouldn't be able to tell. I don't know if the arguments are cleared for the same reason.

**Comment by brenoguim** [comment]
> For your next pull request could you look into makeing a deep copy function for the formatter. We will need it for future refactorings especially with refactoring the scrolling text.

Hi @duncanspumpkin , the `Formatter` today doesn't own the buffer on which it writes to. The deep copy would imply in owning the buffer, right? How would that work?
Maybe I can't understand it because I didn't get to the code that needs that functionality yet.

**Comment by tupaschoal** [comment]
Since it's for a follow-up PR, I'm assuming you can take the conversation to the parent issue @duncanspumpkin and @brenoguim

**Pull request merged**